types/ember/test/lib/assert.ts

```diff
@@ -2,4 +2,4 @@
 // Disable tslint here b/c the generic is used to let us do a type coercion and
 // validate that coercion works for the type value "passed into" the function.
 // tslint:disable-next-line:no-unnecessary-generics
-export declare function assertType<T>(value: T): void;
+export declare function assertType<T>(value: T): T;
```

Comment:
This change facilitates patterns like

```ts
assertType<number>(co8.bar); // $ExpectError
```

Which would otherwise force us to choose between `assertType`'s quick feedback and `$ExpectError`'s ability to work with negative test cases.